docs/tempo/website/getting-started/traces.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
+- /docs/tempo/getting-started/traces
+description: "What are traces?"
+keywords:
+- Grafana
+- traces
+- tracing
+title: What are traces?
+weight: 120
+---
+
+# What are traces?
+
+A trace represents the whole journey of a request or an action as it moves through all the nodes of a distributed system, especially containerized applications or microservices architectures. Traces let you profile and observe systems, making it easy to discover bottlenecks and interconnection issues.
+
+## Example of traces
+
+A user on your website enters their email address into a form to sign up for your mailing list. They click **Enter**. This initial transaction has a trace ID which is subsequently associated with every interaction in the chain of processes within a system.
+
+The user's email address is data that flows through your system. In a cloud computing world, it is possible that clicking that one button causes data to touch multiple nodes across your cluster of microservices.
+
+The email address may be sent to a verification algorithm sitting in a microservice that exists solely for that purpose. If it passes the check, the information is stored in a database.
+
+Along the way, an anonymization node strips personally identifying data from the address and sends metadata collected to a marketing qualifying algorithm to determine whether the request was sent from a targeted part of the internet.
+
+Services respond and data flows back from each, sometimes triggering new events across the system. Along the way, logs are written in various nodes with a time stamp showing when the info passed through.
+
```

Comment:
Should this be in  numbered list? Something about four consecutive two-sentence paragraphs feels a little off.

Reply:
I don't think so because it's not a step by step procedure. But I can update the changes a dsee what comes out.

Reply:
I added some "story" words to keep the flow smooth.

Reply:
Thank you, Ursula.